generator/views.py

```diff
@@ -227,6 +232,8 @@ def get(cls, request, share_id):
         return redirect('/share/' + game.share_id)
 
 
+hashsymbols = base64.b64decode('iVBORw0KGgoAAAANSUhEUgAAAGAAAAAIBAMAAAD0Eec8AAAAD1BMVEUAAAAAngAYICiIkJD4+PicIqvpAAAAAXRSTlMAQObYZgAAANBJREFUGBkFwYGNxAAIA7Ao7SARMMCLY4CKsP9MbyMALKK4AC2dAFRmnoUIBBWcxt01XgEloGKcJ9T15wVQlXPXHGuyzVvY03gF5AL0irOo9FcjsWYc4efu78aXM7Kn727BkTgC8oJ1H71bM1NC1HjT0xW5d9d1JyB3cxeoeiv9oUZl/7jAjJ13XeLa089ZAH8qVSCv6/oD3elblLDZxbFyq+3pZy2gqks88BUtodKsE7jYiKJHHKvOelUCyDW40CcAANkAAI4EkKIQAUaAoP4BJuQ1q9JLjugAAAAASUVORK5CYII=')
```

Comment:
I have to admit I'm a little proud of this -- this data is the following image:

![hashsymbols](https://github.com/Anguirel86/ctjot_web_generator/assets/106004/e25e0253-0d9b-4f27-b132-31b76f8116e8)

png optimization is impressive, to make it so small!